cmd/juno/juno.go

```diff
@@ -84,6 +84,8 @@ const (
 	corsEnableF             = "rpc-cors-enable"
 	versionedConstantsFileF = "versioned-constants-file"
 	pluginPathF             = "plugin-path"
```

Comment:
wdym?  I've added this `logHostF` and `logPortF`

Do you mean a boolean flag to enable/disable the log host?

Reply:
Yeep, but indeed it might be a little bit redundant, but the rest of the configuration includes it. Perhaps it’s better to simplify the others instead